releases/v0.6.0.md

```diff
@@ -0,0 +1,66 @@
+# Release Notes for v0.6.0
+Release Date: June 7th, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+- Support for attesting pod VMs with Azure vTPMs on SEV-SNP
+- Support for using Project Amber as an attestation service
+- Support for Cosign signature validation with s390x
+- Pulling guest images with many layers can no longer cause guest CPU starvation.
+- Attestation Service upgraded to avoid several security issues in Go packages.
+- Several CoCo components are now also shipped in the Kata Containers payload bundle.
```

Comment:
I'd drop this part.

There are no "CoCo components" being shipped in the Kata Containers payload.